src/main/java/com/codegile/univ/mihai/week1/day1/Project1.java

```diff
@@ -0,0 +1,53 @@
+package com.codegile.univ.mihai.week1.day1;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+
+
+public class Project1 {
+
+    /*public void ReadFile(String txt){
+        InputStream inputStream = this.getClass().getResourceAsStream("text.txt");
+        new Scanner(inputStream);
+        System.out.println(inputStream);
+    }*/
+
+    public static void main(String args[]) throws IOException {
+        int nrReale = 0;
+        int nrIntregi = 0;
+
+        File file = new File("resources/day1/text.txt");
+        Scanner scanner = new Scanner(file);
+        String lineSeparator = System.getProperty("line.separator");
```

Comment:
it was good thinking to use the System's line separator, but as you probably realised, it is used anyway by the `nextLine()` functions. Note - there is a `System.getLineSeparator()` method which is more convenient 😉